schema.d.ts

```diff
@@ -102,6 +112,11 @@ export interface JavaScriptExport {
    * defined and the `name` field must be `"*"`.
    */
   declaration: Reference;
+  
+  /**
+   * Is this export deprecated? Provide a string to give a reason or migration instructions.
+   */
+  deprecated?: boolean|string;
```

Comment:
Hmm... What does it mean for an export to be deprecated rather than the thing it's deprecated? If someone deprecates an exported function, do they need to deprecate both?

Reply:
could be they aliased the declaration and deprecated the old name